packages/ui/src/components/Banner/banner.tsx

```diff
@@ -64,12 +66,12 @@ export const Banner: React.FC<BannerProps> = ({
     const textSize = (width >= LG_BARRIER || width < MD_BARRIER) ? 'large' : 'medium'
 
     const handleClick = useCallback((event) => {
-        sendAnalyticsClickEvent('Banner', { title })
+        sendAnalyticsClickEvent('Banner', { title }, id)
```

Comment:
It's more clear to include optional `id` to payload rather than creating separate argument